### ISSUE TITLE
Sqlite ADO.NET Provider - Code First Issue

### ISSUE THREAD

**Issue by Chris-Dickerson** [ErikEJ/SqlCeToolbox#532]
Describe what is not working as expected.

When I attempt to create a code first entity framework from a Sqlite database, sometimes I have this error (and I have to restart Visual Studio and it works as expected):

---------------------------
Microsoft Visual Studio
---------------------------
The system cannot find the path specified.

---------------------------
OK   
---------------------------

I had to re-install Visual Studio due to an issue with an add-in, I thought this issue had gone away... I was able to generate the EF for 2 databases several times before... same thing happened.

### Steps to reproduce
Include a complete description that I can redo to reproduce the issue.

If you point at a Sqlite database, select the "Code First From Database" and randomly (it seems) it will crash (for me at least) when you click finish.

### Further technical details
Toolbox version: (found in About dialog - blue questionmark icon)
v4.7.234.0

Database engine: (SQlite, SQL Compact or SQL Server)
Sqlite 3

Visual Studio or SSMS version: (e.g. Visual Studio 2015, SSMS 16.5.3)
Visual Studio Pro 2017

**Comment by ErikEJ** [comment]
I am not sure this is related to this extension - but have you seen the guide here: https://github.com/ErikEJ/SqlCeToolbox/wiki/Migrate-a-SQL-Server-database-to-SQL-Server-Compact-or-SQLite ?

**Comment by Chris-Dickerson** [comment]
I'm aware of that functionality but it isn't what I was trying to accomplish. I was re-generating the Entity Framework Code First from Database (Sqlite).   I'm not sure who/what is to blame, I'm just seeking answers. :).

**Comment by Chris-Dickerson** [comment]
Figured it out.  

I usually make a folder to put my EF files in for each database. To re-create code first, I need to delete all the files first. I mark them all (shift-click, delete). I've **never** selected the folder itself... somehow Visual Studio itself... has been deleting my folder.  (_The filesystem folder... the one I created from the solution... still there_).

What couldn't be found... was the filesystem folder.

The reason for the randomness I noticed... the folder apparently wasn't getting deleted but every so often...
